src/vs/code/electron-main/window.ts

```diff
@@ -425,11 +431,20 @@ export class VSCodeWindow {
 	}
 
 	private onConfigurationUpdated(config: IConfiguration): void {
-		const newMenuBarVisibility = this.getMenuBarVisibility(config);
+		const newMenuBarVisibility = this.getMenuBarVisibility();
 		if (newMenuBarVisibility !== this.currentMenuBarVisibility) {
 			this.currentMenuBarVisibility = newMenuBarVisibility;
 			this.setMenuBarVisibility(newMenuBarVisibility);
 		}
+
+		// Swipe command support (macOS)
+		const workbenchConfig = this.configurationService.getConfiguration<IWorkbenchEditorConfiguration>();
+		if (workbenchConfig && workbenchConfig.workbench && workbenchConfig.workbench.editor.swipeBetweenOpenFiles) {
```

Comment:
@bpasero I can take that in a different PR if it would be of interest, basically the structure now could technically be changed to a class with optional attributes. It's just a regular object for now(?). I'll leave it and do the checks for now.